### DISCUSSION TITLE
Accidentally reversed playback countdown time?

### DISCUSSION THREAD

**Discussion by v1sages** [cmus/cmus#1232] [Troubleshooting] [answered]
Hi all, not sure if I should open this as an issue, my google-fu isn't working out.
I have attempted to reinstall cmus via `yay -S cmus` (and then fully uninstalling using `yay/pacman -R cmus`, but I can't seem to get around this after a reboot: 

When I play a track from my browser or after loading a music folder, `cmus` just counts DOWN now, instead of starting UP from 0:00. Either I pressed a keybinding I was unaware of or changed something, or cmus has changed this functionality. Is there a ``:set  command I am just missing out on?
**How can I change the countdown back to start a track from 0:00 and go up?**

![121234324235325](https://user-images.githubusercontent.com/112735560/205474465-954c538c-ad03-465a-9364-c945108fa44f.png)

**Comment by gavtroy** [comment, marked as answer]
Hit `t` which is bound to `:toggle show_remaining_time`. To reset everything you would remove the `~/.cmus` directory.
